stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/StorageHelper.kt

```diff
@@ -134,8 +134,11 @@ public class StorageHelper {
         return attachments
     }
 
-    private fun getFileName(attachmentMetaData: AttachmentMetaData): String =
-        "STREAM_${dateFormat.format(Date().time)}_${attachmentMetaData.getTitleWithExtension()}"
+    private fun getFileName(attachmentMetaData: AttachmentMetaData): String {
+        return attachmentMetaData.run {
+            "${title}_STREAM_${dateFormat.format(Date().time)}.${getExtension()}"
```

Comment:
Usually, the title contains a file extension and I think it would look a bit weird with our suffix (test.pdf_STREAM_date.pdf). Maybe we should strip extension from the title?

Reply:
I kept prefix instead of sufix, just removed the `STREAM_` part. And also the item can show multiple lines.

Reply:
Edit: Now the prefix is added in the file, but taken away when the file is shown in the adapter.